include/swift/Frontend/FrontendOptions.h

```diff
@@ -549,8 +541,18 @@ class FrontendOptions {
     return InputKind == InputFileKind::IFK_Swift &&
            Inputs.hasUniqueInputFilename();
   }
+private:
+  const char *computeSuffix();
+  
+  bool canEmitDependencies() const;
+  bool canEmitHeader() const;
+  bool canEmitLoadedModuleTrace() const;
+  bool canEmitModule() const;
+  
+  /// Return true if changed output filename.
```

Comment:
What does this comment have to do with this function, which is clearly a predicate?

Reply:
Oops! Bogus comment. While there, I renamed actionProducesOutput and actionProducesTextualOutput to doesActionProduce[Textual]Output.